Src/DependencyCollector/Shared/Implementation/Operation/CacheProvider.cs

```diff
@@ -3,392 +3,34 @@
     using System;
     using System.Collections.Generic;
     using System.Globalization;
-    using System.Threading;
+    using System.Runtime.Caching;
+    using System.Threading;        
     using Microsoft.ApplicationInsights.Extensibility.Implementation.Tracing;
 
     /// <summary>
-    /// The implementation of cache provider for Windows Phone and Windows Store as MemoryCache is not available there.
+    /// The wrapper for MemoryCache.
     /// </summary>
     /// <typeparam name="TValue">Type of items to store in the cache.</typeparam>
-    internal sealed class CacheProvider<TValue> : ICacheProvider<TValue>
+    internal class CacheProvider<TValue> : ICacheProvider<TValue>
```

Comment:
Right, except we still have all of those specifics, just in an extra wrapper class, which is not reused, and the unit tests are now testing MemoryCache instead of our code which means they should get deleted also.